src/passes/strings-encoding/StringEncoding.cpp

```diff
@@ -269,6 +291,24 @@ bool StringEncoding::injectOnStack(BasicBlock& BB, Instruction& I, Use& Op, Glob
   return true;
 }
 
+std::pair<Instruction *, Instruction *>
+materializeConstantExpression(Instruction *Point, ConstantExpr *CE) {
+  auto *Inst = CE->getAsInstruction();
+  auto *Prev = Inst;
+  Inst->insertBefore(Point);
+
+  Value *Expr = Inst->getOperand(0);
```

Comment:
Taking operand `0` is going to work as long as our assumptions (1) and (2) above hold.

Reply:
As below.